examples/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ if (UMPIRE_ENABLE_HIP)
   blt_add_executable(
     NAME multi_device
     SOURCES multi_device.cpp
-    DEPENDS_ON umpire hip)
+    DEPENDS_ON umpire blt::hip)
```

Comment:
You shouldn't have to list `blt::hip` and `blt::hip_runtime` everywhere that depends on `umpire` because it will be inherited properly.

Reply:
So you would expect for ALL DEPEND_ON hip and hip_runtime to go away in Umpire?

Reply:
Not all but probably most if not all but one.